fluss-common/src/test/java/org/apache/fluss/row/BinaryWriterTest.java

```diff
@@ -124,13 +123,27 @@ public void testValueSetterWithDecimalType() {
     }
 
     @Test
-    public void testCreateValueSetterForMapThrowsException() {
-        assertThatThrownBy(
-                        () ->
-                                createPrimitiveValueWriter(
-                                        DataTypes.MAP(DataTypes.INT(), DataTypes.STRING())))
-                .isInstanceOf(UnsupportedOperationException.class)
-                .hasMessageContaining("Map type is not supported yet");
+    public void testValueSetterWithMapType() {
+        BinaryArray array = new org.apache.fluss.row.array.PrimitiveBinaryArray();
+        BinaryArrayWriter writer = new BinaryArrayWriter(array, 2, 8);
+
+        BinaryWriter.ValueWriter setter =
+                BinaryWriter.createValueWriter(
+                        DataTypes.MAP(DataTypes.INT().copy(false), DataTypes.STRING()),
+                        BinaryRow.BinaryRowFormat.ALIGNED);
```

Comment:
Use `COMPACTED` format in tests by default as it is more widely used than `ALIGNED` and `INDEXED`.